src/main/java/cz/student/simon8491/TaskManager.java

```diff
@@ -1,9 +1,142 @@
 package cz.student.simon8491;
 
+import cz.common.ConsoleColors;
+import org.apache.commons.lang3.ArrayUtils;
+import org.apache.commons.lang3.math.NumberUtils;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+
 public class TaskManager {
+    static final String[] OPTIONS = {"add", "remove", "list", "exit"};
+    private static String[][] tasks;
+    private static Scanner scanner;
+
+    public static void main(String[] args) {
+
+        tasks = loadDataToTab("tasks.csv");
+        displayOptions();
+        scanner = new Scanner(System.in);
+        while (scanner.hasNextLine()) {
+            final var option = scanner.nextLine();
+            switch (option) {
+                case "add" -> addTask();
+                case "remove"-> removeTask();
+                case "list" -> taskList();
+                case "exit" -> {
+                    storeData();
+                    System.out.println("Data were stored successfully");
+                    System.out.println(ConsoleColors.RED + "Sayonara!");
+                    System.exit(0);
+                }
+                default -> System.out.println("Invalid option");
+            }
+            displayOptions();
+        }
+        scanner.close();
+    }
+
+    public static void displayOptions() {
+        System.out.println(ConsoleColors.BLUE);
+        System.out.println("Please select the option for task manager:");
+        for (var option : OPTIONS) {
+            System.out.println(ConsoleColors.BLUE + option);
+        }
+        System.out.println(ConsoleColors.RESET);
+    }
+
+    public static String[][] loadDataToTab(String fileName) {
+        Path file = Paths.get(fileName);
+        String[][] dataTab;
 
-    public static void main(final String[] args) {
+        if (!Files.exists(file)) {
+            System.out.println("File does not exist.");
+            System.err.println("-1");
+            System.exit(0);
```

Comment:
BUG: Pri chybovem stavu se vraci kod 1.

```suggestion
            System.exit(1);
```